README.md

```diff
@@ -6,20 +6,20 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 In the project directory, you can run:
 
-### `yarn start`
+### `npm start`
```

Comment:
- [ ] Please make `README` more professional, you can follow the [microverse README template](https://github.com/microverseinc/readme-template) to make it professional. Please include the title, description, author, and screenshot if you have.

Reply:
Well addressed this!